src/mono/wasm/debugger/DebuggerTestSuite/Tests.cs

```diff
@@ -826,6 +826,24 @@ public async Task GetSourceUsingSourceLink()
             Assert.True(source.IsOk);
         }
 
+        [Fact]
+        public async Task InspectTaskAtLocals() => await CheckInspectLocalsAtBreakpointSite(
+            "InspectTask",
+            "RunInspectTask",
+            7,
+            "<RunInspectTask>b__0" ,
+            $"window.setTimeout(function() {{ invoke_static_method_async('[debugger-test] InspectTask:RunInspectTask'); }}, 1);",
+            wait_for_event_fn: async (pause_location) =>
+            {
+                var locals = await GetProperties(pause_location["callFrames"][0]["callFrameId"].Value<string>());
+
+                var t_arr = await GetObjectOnLocals(locals, "t");
+                await CheckProps(t_arr, new
+                    {
+                        s_taskIdCounter = TNumber(0)
+                    }, "t_arr", num_fields: 53);
```

Comment:
Might be useful to add some property here, like `Task.Status`?

Reply:
added